hw5/task_1.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+
+
+class Homework:
+    def __init__(self, text, deadline):
+        self.text = text
+        self.deadline = datetime.timedelta(deadline)
+        self.created = datetime.datetime.now()
+
+    def is_active(self):
+        return datetime.datetime.now() < (self.created + self.deadline)
+
+
+class Student:
+    def __init__(self, first_name, last_name):
+        self.first_name = first_name
+        self.last_name = last_name
+
+    @staticmethod
+    def do_homework(homework: Homework):
+        if homework.is_active():
+            return homework
+        else:
+            print("You are late.")
+            return None
+
```

Comment:
You no need to code return if you need return None, by default python methods/functions returns None if no return value, python can't havn't return at all.
So if you just print message, it would be enouth for else block